### PR TITLE
Incoming change requests

### PR DESCRIPTION
**Pull request by jbydeley-d2l**
- Toast sends back i18n'ed messages
- postMessage only sends messages back once per exempt/unexempt
- Load More is disabled while loading
- Additional tests
- Clean up destructuring methods
- Additional a11y text added for clarity

### CODE REVIEW

**Comment by RyanSweetD2L** [comment]
:shipit:

File: src/App.vue [hunk 1]
```diff
@@ -1,16 +1,18 @@
 <template>
   <div id="activity-exemptions">
-    <button class="d2l-button primary" @click="setExempt">{{ $t('btnExempt') }}</button>
-    <button class="d2l-button" @click="setUnexempt">{{ $t('btnUnexempt') }}</button>
+    <button :aria-label="$t('ariaExempt')" class="d2l-button primary" @click="setExempt">
```
Comment (lawong-d2l) [inline src/App.vue]: I think we need roles for each element, doesn't have to be done in this PR.

File: src/App.vue [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
     <p>
       {{ $t('lblExemptionCount', {exemptionCount}) }}
     </p>
-    <table>
+    <table :summary="$t('ariaTableSummary')">
       <thead>
         <tr>
           <th>
-            <input type="checkbox" class="d2l-checkbox" :checked="allSelected" @change="selectAll">
+            <input :aria-label="$t('ariaSelectUnselectAll')" type="checkbox" class="d2l-checkbox" :checked="allSelected" @change="selectAll">
```
Comment (lawong-d2l) [inline src/App.vue]: When playing with NVDA, we should add back functionality with aria that when you tab down the list it should say the students' name and their exempt status, so you don't have to press down twice on each row. This was done with row headers before. Could also be in another PR for more a11y improvements.

**Comment by lawong-d2l** [comment]
:shipit:

File: src/store/actions.js [hunk 3]
```diff
@@ -14,56 +15,49 @@ const d2lAxios = axios.create({
 })
 
 export const actions = {
+	/*
+	 * setExempt( context )
+	 * 
+	 * setExempt finds all users that are selected and unexempt and attempts
+	 * to post to the exemptionUpdateURL with the user's ID. A toast message 
+	 * will be
```
[the remark this answers is not in the document]
Comment (jbydeley-d2l) [reply src/store/actions.js]: Oops! I switched from writing this to fixing a bug mid sentence. Comments get stripped during build but I plan on continuing to add more to the code for clarity (and to fix this one).

File: src/App.vue (hunk 1 above)
Comment (jbydeley-d2l) [reply src/App.vue]: We shouldn't, I'm using a `<button>` so it's role is button by default.

File: src/App.vue (hunk 2 above)
Comment (jbydeley-d2l) [reply src/App.vue]: Oh cool! I'll talk to you about how we can add this in later!
